src/lib/Sendbird.tsx

```diff
@@ -128,6 +163,8 @@ const Sendbird = ({
   const [sdkStore, sdkDispatcher] = useReducer(sdkReducers, sdkInitialState);
   const [userStore, userDispatcher] = useReducer(userReducers, userInitialState);
 
+  const { configs, initDashboardConfigs } = useUIKitConfig();
```

Comment:
⚠️ Updated once again after the initial commit ⚠️

To pass `initDashboardConfigs` to useConnect as a param, <UIKitConfigProvider /> should be a parent component of this <Sendbird /> component. 
So instead of using UIKitConfigContext.Consumer inside of this component, I changed the structure of this component like below. 

```
const Sendbird = props => {
  return (
    <UIkitConfigProvider>
      <SendbirdSDK {...props} />
    </UIkitConfigProvider>
  )
}

const SendbirdSDK = props => {
  const { configs, initDashboardConfigs } = useUIKitConfig();
  const reconnect = useReconnect(..., initDashboardConfigs); 
  return (
    <SendbirdSDKProvider {{ ...xxx, reconnect }} />
  )
}
```